ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -95,25 +95,35 @@ mkdir dev-scripts
 tar -xzvf dev-scripts.tar.gz -C /root/dev-scripts
 chown -R root:root dev-scripts
 
-NVME_DEVICE="/dev/nvme0n1"
-if [ -e "\$NVME_DEVICE" ];
+if [ ! -z "${NVME_DEVICE}" ] && [ -e "${NVME_DEVICE}" ];
 then
-  mkfs.xfs -f "\${NVME_DEVICE}"
+  mkfs.xfs -f "${NVME_DEVICE}"
   mkdir /opt/dev-scripts
-  mount "\${NVME_DEVICE}" /opt/dev-scripts
+  mount "${NVME_DEVICE}" /opt/dev-scripts
 fi
 
 cd dev-scripts
 
 cp /root/pull-secret /root/dev-scripts/pull_secret.json
 
-echo "export OPENSHIFT_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE}" >> /root/dev-scripts/config_root.sh
 echo "export ADDN_DNS=\$(awk '/nameserver/ { print \$2;exit; }' /etc/resolv.conf)" >> /root/dev-scripts/config_root.sh
 echo "export OPENSHIFT_CI=true" >> /root/dev-scripts/config_root.sh
 echo "export NUM_WORKERS=3" >> /root/dev-scripts/config_root.sh
 echo "export WORKER_MEMORY=16384" >> /root/dev-scripts/config_root.sh
 echo "export ENABLE_LOCAL_REGISTRY=true" >> /root/dev-scripts/config_root.sh
 
+if [[ "${ARCHITECTURE}" == "arm64" ]]; then
+  echo "export DOCKER_REGISTRY_IMAGE=quay.io/libpod/registry:2.8" >> /root/dev-scripts/config_root.sh
```

Comment:
What is this overriding, and why only on ARM?

Reply:
Went ahead and opened [this](https://github.com/openshift-metal3/dev-scripts/pull/1428)

The image in question will work for the architectures we care about.

Reply:
I'm slightly confused on this comment... Is the suggestion to move the overwrite to all arches, or is this enough clarification that we'll leave it as is?

Reply:
A PR was merged that makes this specific line unnecessary (exporting a new DOCKER_REGISTRY_IMAGE I mean)